### ISSUE TITLE
Upstreaming dependency_support from bazel_rules_hdl

### ISSUE THREAD

**Issue by QuantamHD** [bazelbuild/bazel-central-registry#163]
Hi, 

Also a Googler. I have a pretty extensive set of dependency support package in [bazel_rules_hdl](https://github.com/hdl/bazel_rules_hdl/tree/main/dependency_support) that might be useful to upstream here.

The packages in there are all built hermetically from source, as such, they represent the full DAG of all the packages.

**Comment by fmeum** [comment]
These projects would be a great addition to the BCR. 

I noticed that some projects are patched to use Bazel runfiles. This will definitely lead to issues before https://github.com/bazelbuild/bazel/issues/16124 has been fixed, but that is actively being worked on.

**Comment by meteorcloudy** [comment]
@QuantamHD Yes, we would love to accept PRs to add those dependencies as Bazel modules.

**Comment by freeform-andre** [comment]
https://github.com/bazelbuild/bazel-central-registry/issues/3926 might be related to this depending on what the maintainers plan to do.
